### PR TITLE
Add new config option for collections 

### PR DESCRIPTION
**Pull request by Saira-A**
#### Description of what you did:

Added a new option called "defaultToTreeIfCollection" to the config so that the left panel defaults to Index rather than Thumbnails if the item is a collection, so the tree view of manifests within the collection is visible by default

### CODE REVIEW

**Comment by Saira-A** [comment]
Thanks @demiankatz! I refactored it but I'm not sure about line 286 so let me know if there's a better way :)

**Comment by demiankatz** [comment]
Thanks, @Saira-A! Regarding line 286, I think it would probably make sense to move the remaining logic inside defaultToThumbsView so the logic is consistent. Maybe you could change lines 622-624 to something like:

```
    // Default to tree view if the manifest is a collection and either the "default to tree if collection" setting is
    // on, or the tree has nav dates (which always require a tree):
    if (this.isCollection() && (defaultToTreeIfCollection || this.extension.helper.treeHasNavDates(treeData))) {
      return false;
    }
```

I'm not really familiar with the "nav dates" functionality, though, so I'm basing this suggestion on a logical refactoring of the code more than on direct experience with the intent behind the code... so please take my suggestion with the appropriate grain of salt.

File: src/content-handlers/iiif/extensions/config/ContentLeftPanel.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ type ContentLeftPanelOptions = ExpandPanelOptions & {
   defaultToTreeEnabled: boolean;
   /** Number of items to default to tree view */
```
Comment (demiankatz) [inline src/content-handlers/iiif/extensions/config/ContentLeftPanel.ts:22]:
```suggestion
  /** Number of items to default to tree view (when defaultToTreeEnabled = true; defaults to 0) */
```

File: src/content-handlers/iiif/extensions/config/ContentLeftPanel.ts [hunk 2]
```diff
@@ -21,6 +21,8 @@ type ContentLeftPanelOptions = ExpandPanelOptions & {
...
   defaultToTreeIfGreaterThan: number;
+   /** Determines if collection should default to tree view */
```
Comment (demiankatz) [inline src/content-handlers/iiif/extensions/config/ContentLeftPanel.ts:24]:
```suggestion
   /** Determines if collection should default to tree view (even if defaultToTreeEnabled = false) */
```

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @Saira-A! This looks good to me now (though I haven't personally tested it); see below for some suggestions that might help clarify the relationships between the configuration settings. Feel free to accept, modify or ignore as you see fit. :-)

**Comment by Saira-A** [comment]
Makes sense, thanks @demiankatz!

**Review by demiankatz — APPROVED** [review]
Thanks, @Saira-A, I'm approving this so merging won't be blocked and because the logic makes sense to me -- but it would probably be a good idea to get some hands-on testing on this before merging it if anyone has time (assuming @LanieOkorodudu hasn't already done it). :-)

**Comment by LanieOkorodudu** [comment]
> Thanks, @Saira-A, I'm approving this so merging won't be blocked and because the logic makes sense to me -- but it would probably be a good idea to get some hands-on testing on this before merging it if anyone has time (assuming @LanieOkorodudu hasn't already done it)

I just test it and it should be fine to merge @demiankatz

**Comment by demiankatz** [comment]
Thanks, @LanieOkorodudu!

**Pull request merged**